src/App.tsx

```diff
@@ -37,10 +38,14 @@ const footerLinks: DataSource[] = [
 function App() {
   return (
     <IntlProvider locale={userLocale} messages={messages[userLocale]}>
-      <div className="relative isolate h-full min-h-screen">
-        {/* Sidebar  */}
-        <Sidebar {...{ footerLinks, dataSources }} />
-        <div className=""></div>
+      <div className="flex justify-between">
+        <div className="relative isolate h-full min-h-screen">
+          {/* Sidebar  */}
+          <Sidebar {...{ footerLinks, dataSources }} />
+        </div>
+        <div className="-z-10 h-full w-full">
```

Comment:
We had a discussion around _not_ using z-index for this. Did you try without using `z-index`?